### PR TITLE
docs: remove outdated notes on splits from roadmap

### PR DESCRIPTION
**Pull request by JKillian**
**What this PR does / why we need it**:
Removes some outdated notes from the roadmap about VSCode window limitations that were no longer new.

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -439,8 +439,8 @@ Miscellanea:
 | :white_check_mark:        | :1234: @@                         | repeat previous @{a-z} (N times)                                                                   |
 | :arrow_down:              | :@{a-z}                           | execute the contents of register {a-z} as an Ex command                                            |
 | :arrow_down:              | :@@                               | repeat previous :@{a-z}                                                                            |
-| :arrow_down:              | :[range]g[lobal]/{pattern}/[cmd]  | execute Ex command [cmd](default: ':p') on the lines within [range] where {pattern} matches        |
-| :arrow_down:              | :[range]g[lobal]!/{pattern}/[cmd] | execute Ex command [cmd](default: ':p') on the lines within [range] where {pattern} does NOT match |
+| :arrow_down:              | :[range]g[lobal]/{pattern}/[cmd]  | execute Ex command [cmd](default: ":p") on the lines within [range] where {pattern} matches        |
+| :arrow_down:              | :[range]g[lobal]!/{pattern}/[cmd] | execute Ex command [cmd](default: ":p") on the lines within [range] where {pattern} does NOT match |
```
Comment (JKillian) [inline ROADMAP.md]: this change was automatically introduced by prettier

File: test/state/vimState.test.ts [hunk 2]
```diff
@@ -17,10 +17,7 @@ suite('VimState', () => {
     const vimState = new VimState(vscode.window.activeTextEditor!);
     const cursorStart = new Position(0, 0);
     const cursorStop = new Position(0, 1);
-    const initialCursors = [
-      new Range(cursorStart, cursorStop),
-      new Range(cursorStart, cursorStop)
-    ];
+    const initialCursors = [new Range(cursorStart, cursorStop), new Range(cursorStart, cursorStop)];
```
Comment (JKillian) [inline test/state/vimState.test.ts]: `gulp forceprettier` seems to have made a few unrelated changes, I left them in since it seemed harmless

**Comment by jpoon** [comment]
Thanks @JKillian
